packages/graphql-typescript-definitions/test/document.test.ts

```diff
@@ -1,5 +1,6 @@
 import * as path from 'path';
 
+// eslint-disable-next-line @shopify/typescript/prefer-build-client-schema
```

Comment:
The [migration to fix this seems pretty straightfoward](https://github.com/Shopify/web-configs/blob/dab62e4f72e14c1f33ae6a6a626f890e2a872f3e/packages/eslint-plugin/docs/rules/typescript/prefer-build-client-schema.md). are we able to do it?